routers/posts.py

```
@@ -0,0 +1,29 @@
+from fastapi import APIRouter, status, Query
+from schemas.post import PostCreate, PostUpdate
+from typing import Optional
+
+router = APIRouter(prefix="/posts",tags=["Likes"])
+
+@router.get("") # 목록 조회, 검색, 정렬 포함
+def get_posts(page: int = 1, search: Optional[str] = None, sort: str = "latest"):
+    return {"message": f"{page}페이지 {sort}순 목록 조회 (검색어: {search})"}
+
+@router.post("", status_code=status.HTTP_201_CREATED)
+def create_post(data: PostCreate):
```

Ruff (0.14.11)

Unused function argument: `data`

(ARG001)